foundation/okfplugins/feature_block/migrations/0009_alter_featureblock_block_type.py

```diff
@@ -17,7 +15,7 @@ class Migration(migrations.Migration):
                     ("yellow_box", "Yellow Box"),
                     ("white_box", "White Box"),
                     ("transparent_title", "Transparent with Title"),
-                    ("transparent", "Transparent"),
+                    ("transparent_no_title", "Transparent without Title"),
```

Comment:
In Django, you don't change old migrations. They were already applied in the database.
We expect a new migration when a model changes.
I'm not sure why we need to change a previously applied migration

Reply:
I did not change the migration directly. But had to touch that file while fixing conflict. What do you suggest to do in such situation? Thank you!

Reply:
In this case, I think you should:
 - Undo this change in your computer (to let this file unchanged)
 - Remove the files that depends on this migration (just `0010_featureblock_text_color`)
 - Run the `makemigrations` command again.

This should regenerate the migrations properly.
You can commit those changes.

I don't see any conflict right now and doing this you will not get any conflict (I expect).

Reply:
Thank you!. I restored the direct changes to the migration as you had suggested. However I did not delete this file `0009_alter_featureblock_block_type.py` because of the following
- there was chain of files dependent on each other
- I removed all (dependent ones) and regenerated migration. 
It did not get any conflict but the new migration generated another migration file which was the update to the existing file(same as before).

So, I only restored the main file which was changed during merge conflict `0009_alter_featureblock_block_type.py`

It will not cause any issue because there is another file with similar name which has those changes.
`0010_alter_featureblock_block_type.py`

Reply:
Maybe I explained myself wrong.
I wasn't referring to deleting the `0009_alter_featureblock_block_type` file, just leaving it as it was in its previous version.

It looks good now